corehq/apps/settings/forms.py

```diff
@@ -54,8 +56,15 @@ def __init__(self, user, *args, **kwargs):
             ),
         )
 
-    def clean_new_password1(self):
-        return clean_password(self.cleaned_data.get('new_password1'))
+    def clean_old_password(self):
+        from corehq.apps.hqwebapp.utils import decode_password
+        old_password = decode_password(self.cleaned_data['old_password'])
+        if old_password == '':
+            raise ValidationError(
+                _("Password cannot be empty"), code='password_empty',
+            )
```

Comment:
> This check would only be true in case the encoding is incorrect/messed up

Can a normal (not trying to do anything sneaky) user trigger this condition? I think we should not be adding code to give nice/better messages to people who are messing with the obfuscation.